### ISSUE TITLE
Wrong signature in model call in run_classifier.py example (?)

### ISSUE THREAD

**Issue by davidefiocco** [huggingface/pytorch-pretrained-BERT#76]
I think that
https://github.com/huggingface/pytorch-pretrained-BERT/blob/063be09b714bf4d2fbbc3de7f52c45b8bc6817eb/examples/run_classifier.py#L608

may well have a problem, as it's not consistent with
https://github.com/huggingface/pytorch-pretrained-BERT/blob/063be09b714bf4d2fbbc3de7f52c45b8bc6817eb/examples/run_classifier.py#L549
nor with
https://github.com/huggingface/pytorch-pretrained-BERT/blob/063be09b714bf4d2fbbc3de7f52c45b8bc6817eb/pytorch_pretrained_bert/modeling.py#L875

and this currently breaks the example.
One quick patch would be to replace that line with 

```
tmp_eval_loss = model(input_ids, segment_ids, input_mask, label_ids)
logits = model(input_ids, segment_ids, input_mask)
```

But I am not so sure, there are likely better ways.

**Comment by yanshanjing** [comment]
You are right, I also encountered this small error.

**Comment by thomwolf** [comment]
Thanks for noticing, fixed in #77.
